frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -135,52 +136,42 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
+                                        <p><b>Display Peers Seen Hijack:</b></p>
                                     </div>
                                 </div>
                                 <div class="row">
-                                    <div class="col-lg-6 offset-lg-6">
-                                          <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#seenHijackWithdraw" aria-expanded="false" aria-controls="seenHijackWithdraw">Seen Hijack BGP Withdrawal</button>
+                                    <div class="col-lg-4">
+                                        <button class="btn btn-info" type="button" data-toggle="collapse" data-target="#seenHijackUpdate" aria-expanded="false" aria-controls="seenHijackUpdate">BGP Announcement</button>
+                                    </div>
+                                    <div class="col-lg-4">
+                                          <button class="btn btn-info" type="button" data-toggle="collapse" data-target="#seenHijackWithdraw" aria-expanded="false" aria-controls="seenHijackWithdraw">BGP Withdrawal</button>
                                     </div>
                                 </div>
                             </div>
                         </div>
                     </div>
                 </div>
             </div>
-            {% if current_user.has_roles('admin') %}
             <div class="col-lg-3">
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <i class="fa fa-bell fa-fw"></i> Actions
-                        {% if current_user.has_roles('admin') %}
-                        <button type="button" id="hijack_acknowledged_button" class="btn btn-primary btn-sm float-right hijack-seen-button">Mark Hijack as <b>acknowledged</b></button>
-                        {% endif %}
-                    </div>
-                    <div class="panel-body">
-                        <div class="row">
-                            <div class="col-lg-4 ">
-                                <button id="resolved_button" type="button" class="btn btn-success" disabled="true">Resolve</button>
-                            </div>
-                            <div class="col-lg-4">
-                                {% if mitigate == True %}
-                                <button id="mitigation_button" type="button" class="btn btn-danger" disabled="true">Mitigate</button>
-                                {% endif %}
-                            </div>
-                            <div class="col-lg-4">
-                                <button id="ignored_button" type="button" class="btn btn-warning" disabled="true">Ignore</button>
+                {% if current_user.has_roles('admin') %}
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <i class="fa fa-bell fa-fw"></i> Hijack Actions
```

Comment:
what are these fa, fw, etc.?

Reply:
Its reserved for icons using font-awesome